packages/genesys-spark-components/src/components/beta/gux-tooltip-beta/gux-tooltip.tsx

```diff
@@ -72,10 +76,34 @@ export class GuxTooltip {
     trackComponent(this.root, { variant: this.placement });
   }
 
+  componentDidLoad(): void {
+    if (!this.tooltipObserver && window.MutationObserver) {
+      this.tooltipObserver = new MutationObserver(
+        (): string => (this.role = this.baseTooltip?.role)
+      );
+    }
+
+    if (this.tooltipObserver) {
```

Comment:
Would everything still work if we moved lines 87-94  below line 84 so it's within the first `if` block in `componentDidLoad`, instead of having separate `if` blocks?

Reply:
grouped together now.